### ISSUE TITLE
关于 | icyfish's blog

### ISSUE THREAD

**Issue by icyfish** [icyfish/blog-comments#5]
https://icyfish.me/about/

**Comment by icyfish** [comment]
## Hello World

**Comment by time-river** [comment]
^ ^

**Comment by magicdawn** [comment]
厉害 围观 ...

**Comment by buf1024** [comment]
加油！

**Comment by magicdawn** [comment]
啊哈~居然是 issue

**Comment by XueRainey** [comment]
哇，我好想认识博主，博主会给机会么。

**Comment by time-river** [comment]
😂这一条issue让我们每个人都收到了On May 24, 2017 4:59 PM, Rainey <notifications@github.com> wrote:哇，我好想认识博主，博主会给机会么。

**Comment by XueRainey** [comment]
以后每一个评论大家都会受到邮件了么，感觉用issues做评论系统的思路有点清奇啊，万一哪天我表白楼主，那不是一群人围观？

**Comment by time-river** [comment]
对。除非手动在GitHub上关了。记得v站上是有这种评论的，因多说的关闭。On May 24, 2017 5:09 PM, Rainey <notifications@github.com> wrote:以后每一个评论大家都会受到邮件了么，感觉用issues做评论系统的思路有点清奇啊，万一哪天我表白楼主，那不是一群人围观？

**Comment by liaoyuqin** [comment]
哈哈哈，表白博主！真的能收到吗？

**Comment by Cloverii** [comment]
好巧好巧，跟妹子用的一样的博客主题～想问下在中文下使用英文标点是出于什么考虑呢？

**Comment by Ding-Fan** [comment]
@Cloverii 出于个人习惯的考虑（我猜）。使用中、英文标点，使用全、半角字符，是否在汉字与其它字符之间加空格，都是基于个人习惯的排版方式。

**Comment by icyfish** [comment]
@Cloverii 也是开始写代码之后的习惯, 经常会中英文混打, 特别是写博客的时候经常会出现这种情况, 为了方便, 还有防止代码里不小心出现中文标点, 就这样了. @Ding-Fan 你好认真, 哈哈. 其实跟排版没什么关系啦, 中文用中文标点排版应该会更好看吧, 就是我比较懒, 贪方便. 😁

**Comment by Cloverii** [comment]
@Ding-Fan 是否在汉字与其他字符之间加空格我倒是觉得应该是排版或者渲染的系统做的事情，虽然因为我的博客主题并没有这种效果所以我是手动加的空格……反正我是句末中文就加全角标点，句末英文就加半角标点。其实我倒是觉得全用中文标点更好看，不过感觉不合适

**Comment by hjdtl** [comment]
文章里为什么不能评论，提示Comments Not Initialized。其实我想问一下，每次修改js文件，是不是每次都要手打一遍webpack?求解答~

**Comment by icyfish** [comment]
@hjdtl 谢谢提醒, 我忘记初始化了, 不好意思. 我有点不太明白你的问题, 可以说得详细一点吗? 现在应该可以到文章下面评论啦.

**Comment by hjdtl** [comment]
就是每次修改代码，之后都要输入指令webpack，用来更新bundle.js文件。如果没有输入指令webpack，bundle是不会更新的。所以每一次修改都要打一次webpack，这样操作对吗？

**Comment by XueRainey** [comment]
@hjdtl webpack只是一个打包工具，用来合并代码，通过各种插件可以实现ES6的编译、LESS和SASS等动态样式语言、丑化压缩代码等等一系列工作流。你的问题是如果JS改变以后要不要执行webpack，当你的源码更改以后，需要使用webpack打包生成一份生产环境下的代码以供使用。 @icyfish 所提到的webpack-dev-server是一个可以热更新代码的服务，可以无需手动刷新网页，通过打补丁的方式实现网页的热更新。还有一种[-watch]命令，可以监听你代码的变化从而实现源代码的自动编译。

**Comment by hjdtl** [comment]
@XueRainey @icyfish 谢谢，我想watch就是我想要的，感谢感谢~

**Comment by mike652638** [comment]
Hi icyFish, 我在GitHub上搜到你的 :)
我也是英语转WEB前端的, 爱自学, 爱折腾, 希望能认识你哈,
我的GitHub: https://github.com/mike652638
现在FCC刷完了, 小项目页: https://www.mike652638.com/demo.html
